### DISCUSSION TITLE
How to find out ((no paths)) and (hidden) section information using rom_report

### DISCUSSION THREAD

**Discussion by RamPingle-eaton** [zephyrproject-rtos/zephyr#73087] [General]
Dear Team, 

We are in the process of transitioning from Zephyr version 3.2.0 to Zephyr 3.6.0. To understand the implications of this transition on our application’s flash memory usage, I have utilized the “west build -r rom_report” command. This has allowed us to generate a report comparing the memory usage of our application developed on both versions of the operating system (3.2.0 vs 3.6.0).

For your reference, I have uploaded the rom_3.6.json file, from which some sections have been removed to reduce its size. Upon review, I have noticed two sections as follows:

{
    "name": "(no paths)",
    "size": 99081
}
and 
{
    "name": "(hidden)",
    "size": 616076
}

I kindly request your assistance in obtaining more information about these (no path) and (hidden) sections. This will greatly aid us in understanding and potentially reducing the memory usage of the ELF file.

Thank you for your attention to this matter.

[rom_3.6_tickt.json](https://github.com/zephyrproject-rtos/zephyr/files/15379603/rom_3.6_tickt.json)

**Comment by epc-ake** [comment]
I'm also interested in this. 
Compiling the WIFI sample for the esp32s3_devkitm results in 154kB (55%) hidden ram usage!
